include/util/guidance/name_announcements.hpp

```diff
@@ -27,8 +27,8 @@ namespace guidance
 // Name Change Logic
 // Used both during Extraction as well as during Post-Processing
 
-inline util::StringView longest_common_substring(const util::StringView &lhs,
-                                                 const util::StringView &rhs)
+inline std::string_view longest_common_substring(const std::string_view &lhs,
```

Comment:
nit. it seems that it is better to pass `string_view` by reference https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/

We had reference here already, but as we are changing it anyway...